### PR TITLE
Update user-submission.md

### PR DESCRIPTION
**Pull request by BrunoDal**
Update for user reporting only for phish &  report in user quarantine.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/user-submission.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ After you've verified that your mailbox meets all applicable prerequisites, you
      - **User reporting experience section**
        - **Before reporting** tab: In the **Title** and **Message body** boxes, enter the descriptive text that users see before they report a message using the Report Message add-in or the Report Phishing add-in. You can use the variable %type% to include the submission type (junk, not junk, phish, etc.).
        - **After reporting** tab: In the **Title** and **Confirmation message** boxes, enter the descriptive text that users see after they report a message using the Report Message add-in or the Report Phishing add-in. You can use the variable %type% to include the submission type.
+       -  **Only display when user reports phishing** : Check this option if you want to display the message only when an email is reporeted as phish, if not checked messages will be shown for any kind of reports.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/user-submission.md]:
```suggestion
       -  **Only display when user reports phishing**: Check this option if you want to display the message only when an email is reported as phish. If not, checked messages will be shown for any kind of report.
```

File: microsoft-365/security/office-365-security/user-submission.md [hunk 2]
```diff
@@ -118,6 +119,10 @@ After you've verified that your mailbox meets all applicable prerequisites, you
    - **Microsoft Outlook Report Message button** \> **Off** ![Toggle off.](../../media/scc-toggle-off.png): Select this option if you use third-party reporting tools instead of the Report Message add-in, the Report Phishing add-in, or the built-in reporting in Outlook on the web, and then configure the following settings:
      - Select **Use this custom mailbox to receive user reported submissions**. In the box that appears, enter the email address of an existing Exchange Online mailbox that can receive email.
 
+- **Microsoft Outlook Report Message button** 
+
+Enable this feature if you want to let end users report a message from the user quarantine.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/user-submission.md]:
```suggestion
   - **Microsoft Outlook Report Message button**: Enable this feature if you want to let end users report a message from the user quarantine.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @BrunoDal, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Comment by siosulli** [comment]
@Dansimp @chrisda - the update here is:

"- **Microsoft Outlook Report Message button**: Enable this feature if you want to let end users report a message from the user quarantine." 

I don't want to just merge this as 'user quarantine' is not something I am familiar with so if could review and merge that would be great. Thanks.

**Comment by chrisda** [comment]
Yeah, there really is no such thing as 'user quarantine'; it's just quarantine. Users only see their own messages (where they're a recipient). Admins see everyone's messages, and often more types of quarantined messages (malware, quarantined by transport rules, etc.). Same quarantine URL for everyone.

**Pull request merged**